beamer/tests/contracts/test_request_manager_fees.py

```diff
@@ -79,16 +82,16 @@ def test_fee_gas_price_updatable_by_owner(deployer, request_manager, token):
     make_request(request_manager, token, requester, requester, 23, zero_fees=False)
 
     old_gas_price = request_manager.gasPrice()
-    old_fee = request_manager.totalFee()
+    old_fee = request_manager.baseFee()
 
     new_gas_price = old_gas_price * 2
 
     with brownie.reverts("Ownable: caller is not the owner"):
-        request_manager.updateFeeData(new_gas_price, 45_000, {"from": requester})
+        request_manager.updateFeeData(new_gas_price, 45_000, 1_000, {"from": requester})
 
-    request_manager.updateFeeData(new_gas_price, 45_000, {"from": deployer})
+    request_manager.updateFeeData(new_gas_price, 45_000, 1_000, {"from": deployer})
     assert request_manager.gasPrice() == new_gas_price
-    assert request_manager.totalFee() == 2 * old_fee
+    assert request_manager.baseFee() == 2 * old_fee
 
 
 def test_fee_reimbursed_on_expiration(request_manager, token):
```

Comment:
Maybe test that this reimbursement calculation is still correct if the fees are updated to a different value in between request made and request withdrawn.

Reply:
That's a very good suggestion, thanks!